index.html

```diff
@@ -198,6 +198,55 @@ <h2>Skills</h2>
           <div class="workicon3"></div>
         </div>
       </section>
+      <section id="form">
+          <nav>
+            <div class="menu">
+              <div class="shape">
+                <div id="triangle"></div>
+                <div id="diamond"></div>
+              </div>
+            </div>
+          </nav>
+          <div class="form-title">
+          <h1 class="title">Get started</h1>
+          <p class="paragraph">
+            Hello! I am a software developer! I can help you build a product,
+            feature or website. Fill this form and lets work.
+          </p>
+          <form action="/my-handling-form-page" method="post">
+            <div class="form-wrapper">
+
+              <div class='full-input'><label for='name'> Your name</label>
+                <input class="input" id="name" type='text' name='name' required placeholder="Anastasiia"></input></div>
+              
+              <div class='full-input'><label for='email'> Your email</label>
+                <input class="input" id="email" type='email' name='email'></input></div>
```

Comment:
- [ ] Kindly add `required` attribute to this input field for validation reason.